### PR TITLE
show threads

### PR DESCRIPTION
**Pull request by sw-double**
https://github.com/radio-t/radio-t-site/issues/94

Screenshot 2019-06-17 at 13 33 11

@Reeywhaar

### CODE REVIEW

**Comment by Reeywhaar** [comment]
Wow, thank you so much!

At first glimpse I see problem: Comments on level 5 and below should not be indented (think of mobile), yet somehow show that they are replies 🤔

**Comment by umputun** [comment]
I think the line should be aligned to the left side of the first symbol. I know Reddit does it in the middle of the first one, but they have arrows here. 

>  Comments on level 5 and below should not be indented (think of mobile), yet somehow show that they are replies 🤔

I think the current indication on a deep level is fine but will be nice to fix it to actually jump to the parent comment

wbfcn-201906-17111237-qw6g2

**Comment by umputun** [comment]
this is an example as NYT shows threads
7of6y-201906-29115551-hid8w

**Comment by sw-double** [comment]
fixed alignment

**Comment by umputun** [comment]
@sw-double & @Reeywhaar - what is the status of PR? is it still WIP or good to be merged?

**Comment by sw-double** [comment]
not sure that it handles dark theme in an idiomatic way

otherwise LGTM on my end

**Comment by umputun** [comment]
@sw-double cool, thx!

@Reeywhaar - need your blessing

**Comment by Reeywhaar** [comment]
I'm working on ui and stream api, plan to end in next 3 hours, then I'll watch this PR.
@umputun hope you don't rush to make it before the show :-)
@sw-double, thank you so much 👍

**Comment by umputun** [comment]
sure, take your time. thx

File: frontend/app/components/comment/_level/comment_level.scss [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-          @media (-moz-touch-enabled: 1) and (max-width: 768px), (pointer: coarse) and (max-width: 768px) {
-            margin-left: calc(-$i * $step);
-          }
-        }
-      }
-    }
-  }
+.thread > .thread {
+  padding-left: 17px;
+  border-left: 1px solid rgba(0, 0, 0, 0.15);
```
Comment (Reeywhaar) [inline frontend/app/components/comment/_level/comment_level.scss]: Is it me, or lines color is too active in light theme?

Screenshot 2019-06-30 at 02 14 01

I'd rather mute it to rgba(0,0,0,.07)
Screenshot 2019-06-30 at 02 17 33

... or even to rgba(0,0,0,.05)
Screenshot 2019-06-30 at 02 14 15

... or another variant to make border dotted:
Screenshot 2019-06-30 at 02 13 29

File: frontend/app/components/comment/_level/comment_level.scss [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
+}
+.thread_level_4 .thread > .thread {
+  padding-left: 5px;
+}
+.comment_theme_dark ~ .thread {
```
Comment (Reeywhaar) [inline frontend/app/components/comment/_level/comment_level.scss]: I'd rather move this selector to `comment_theme_dark.scss` and make it like 
```
.comment_theme_dark {
  ...
  & ~ .thread {...}
}

**Comment by Reeywhaar** [comment]
In general I see no crimes here, concise work, @sw-double, thank you, once again! So, @umputun, if you like it then great, I can fix some minor issues myself.

**Comment by Reeywhaar** [comment]
I think we should make something with mobile layout
Screenshot 2019-06-30 at 02 36 32

**Comment by umputun** [comment]
Generally, I like it, thx @sw-double . Two issues to address:

1. All internals "levels" don't need any visible thread lines. On such a deep level it is already impossible to understand the tree and adding zebra doesn't make it any better
1. I agree with @Reeywhaar - those lines are too visible. Your dotted variant looks better to me

4wxbw_20190629_191748 a4ra6

**Comment by Reeywhaar** [comment]
I'm afraid without the lines it becomes even more unclear
Screenshot 2019-06-30 at 03 50 06

... I can make level5+ lines even more muted
Screenshot 2019-06-30 at 03 52 04

**Comment by umputun** [comment]
Is there a reason why 5+ level indentations are any different? Or why we even have any for such deep levels?

**Comment by Reeywhaar** [comment]
Not sure I follow. 5+ are indented less mainly for mobile and for big discussions. In old version, as you may remember, levels 5+ were unindented at all. Dunno what helps there actually, I think comments this deep are mess no matter what. What do you suggest?

**Comment by Reeywhaar** [comment]
One another solution is to set min-width on thread, and if some discussiton breaks this limit then we get horizontal scroll
ezgif-2-1289a7ea05e1

**Comment by umputun** [comment]
My suggestion is to revert 5+ to unindented

**Comment by Reeywhaar** [comment]
ok, agree

**Pull request merged**